version.txt

```diff
@@ -52,4 +52,4 @@ veg_plots_sdl.json,1.1.1
 vertnet.py,1.3.0
 wine_composition.json,1.1.1
 wine_quality.json,1.1.1
-wood_density.py,1.2.1
+wood_density.py,1.2.1
```

Comment:
OK, thanks. The newline doesn't matter, so just make sure to rerun `version.py` after you've updated the version number and everything should be fine.